test/infrastructure/docker/templates/cluster-template-development.yaml

```diff
@@ -98,6 +98,7 @@ spec:
     spec:
       joinConfiguration:
         nodeRegistration:
+          criSocket: unix:///var/run/containerd/containerd.sock
```

Comment:
maybe we could add a comment why we have to set criSocket here, because other providers won't be required to do this?

Reply:
This is required because kubeadm tries to detect the runtime. If it finds a docker and a containerd socket it will give precedence to docker. A docker socket is detected when docker is mounted to the container. This is a problem because the kindest/node container only ships with containerd.